src/classes/Lingohub/Content.php

```diff
@@ -73,7 +73,7 @@ private function resolveTranslatableContent(array &$obj, array $fields, string $
             if (!isset($fields[$key])) {
                 continue;
             }
-            if (!$fields[$key]['translate']) {
+            if (!isset($fields[$key]['translate']) || !$fields[$key]['translate']) {
```

Comment:
I don't think that's correct. If `translate: false` is set, the translation should be skipped. But in every other case, the field should be translated (similar to `translate`).

AFAIK, this should be:
```suggestion
            if (!isset($fields[$key]['translate']) && !$fields[$key]['translate']) {
```